### PR TITLE
Merge AzureRM.Automation into preview/release

### PR DESCRIPTION
**Pull request by MiYanni**
## Description

This is to merge the Automation changes from the last couple months into our primary branch. The issues from the Automation preview releases have been addressed (missing tests and broken cmdlet).

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by MiYanni** [comment]
@azuresdkci Retest this please

File: src/ResourceManager/Automation/Commands.Automation/AzureRM.Automation.psd1 [hunk 1]
```diff
@@ -164,10 +177,10 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = '* Renamed cmdlet DLL filename to Microsoft.Azure.Commands.Automation.dll'
+        ReleaseNotes = '* Fixed the DSC Register Node command'
 
         # Prerelease string of this module
-        # Prerelease = ''
+        Prerelease = 'preview'
```
Comment (cormacpayne) [inline src/ResourceManager/Automation/Commands.Automation/AzureRM.Automation.psd1]: @maddieclayton 👀

File: tools/AzureRM/AzureRM.psd1 [hunk 2]
```diff
@@ -57,7 +57,7 @@ RequiredModules = @(@{ModuleName = 'AzureRM.Profile'; RequiredVersion = '5.8.0';
                @{ModuleName = 'Azure.AnalysisServices'; RequiredVersion = '0.5.4'; }, 
                @{ModuleName = 'AzureRM.ApiManagement'; RequiredVersion = '6.1.5'; }, 
                @{ModuleName = 'AzureRM.ApplicationInsights'; RequiredVersion = '0.1.8'; }, 
-               @{ModuleName = 'AzureRM.Automation'; RequiredVersion = '5.1.2'; }, 
+               # @{ModuleName = 'AzureRM.Automation'; RequiredVersion = '5.1.2'; },
```
Comment (cormacpayne) [inline tools/AzureRM/AzureRM.psd1]: @maddieclayton 👀

File: src/ResourceManager/Automation/Commands.Automation/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -43,8 +43,8 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 
-[assembly: AssemblyVersion("5.1.2")]
-[assembly: AssemblyFileVersion("5.1.2")]
+[assembly: AssemblyVersion("6.0.1")]
+[assembly: AssemblyFileVersion("6.0.1")]
```
Comment (cormacpayne) [inline src/ResourceManager/Automation/Commands.Automation/Properties/AssemblyInfo.cs]: @maddieclayton 👀

**Pull request merged**